web_searchbar_full_width/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```

Comment:
Link is wrong:

# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).